source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluationCalificationsForEmployeeManager.cs

```diff
@@ -0,0 +1,60 @@
+﻿using CommonJobs.Application.EvalForm.Dtos;
+using CommonJobs.Application.Evaluations.EmployeeSearching;
+using CommonJobs.Domain.Evaluations;
+using CommonJobs.Infrastructure.RavenDb;
+using System;
+using System.Linq;
+
+namespace CommonJobs.Application.EvalForm.Commands
+{
+    public class GetEvaluationCalificationsForEmployeeManager : Command<CalificationsDto>
+    {
+        private string _period;
+
+        private string _evaluatedUser;
+
+        public GetEvaluationCalificationsForEmployeeManager(string period, string evaluatedUser)
+        {
+            _period = period;
+
+            _evaluatedUser = evaluatedUser;
+        }
+
+        public override CalificationsDto ExecuteWithResult()
+        {
+            // The employeeManager should be able to see any evaluation that has been finished.
+
+            var evId = EmployeeEvaluation.GenerateEvaluationId(_period, _evaluatedUser);
+            var evaluation = RavenSession.Load<EmployeeEvaluation>(evId);
+
+            if (evaluation == null)
+            {
+                throw new ApplicationException(string.Format("Error: Evaluación inexistente: {0}.", evId));
+            }
+
+            if (!evaluation.Finished)
+            {
+                throw new ApplicationException(string.Format("Error: La evaluación: {0} debe estar finalizada para poder ser vista.", evId));
+            }
+
+            Employee_Search.Projection employee = RavenSession
+                .Query<Employee_Search.Projection, Employee_Search>()
+                .Where(x => x.IsActive && x.UserName == evaluation.UserName).FirstOrDefault();
+
+            var califications = RavenSession
```

Comment:
`califications` is an IQueriable, right? so, it is being evaluated in lines [48](https://github.com/CommonJobs/CommonJobs/pull/447/files#diff-8e4041f8bbd301a1e4ff6106fe6c36b6R48) and [56](https://github.com/CommonJobs/CommonJobs/pull/447/files#diff-8e4041f8bbd301a1e4ff6106fe6c36b6R56). 

If I am not wrong, you are doing two different queries to database, it is not necessarily wrong, but, could you do it more explicit?

For example evaluating both queries just after this line.